scripts/install/opt/seagate/cortx/motr/conf/support.yaml

```diff
@@ -1,4 +1,2 @@
-
-bundle:
-    create:
-      url: "sh /opt/seagate/cortx/motr/libexec/m0reportbug-bundler"
+support_bundle:
+    - "sh /opt/seagate/cortx/motr/libexec/m0reportbug-bundler"
```

Comment:
Why is it an array?
Why not jus key:val ?

Reply:
@huanghua78 , this is not a final version of support.yaml.
This is stopgap arrangements, so that support bundle is getting generated. Currently YAML parsing of setup.yaml is failing due to changes made for "upgrade". Hence we are asking to have the same entry (like setup.yaml) in support.yaml
We will let you know once new format is ready for following.